pyqtgraph/__init__.py

```diff
@@ -59,6 +59,7 @@
     'exitCleanup': True,    ## Attempt to work around some exit crash bugs in PyQt and PySide
     'enableExperimental': False, ## Enable experimental features (the curious can search for this key in the code)
     'crashWarning': False,  # If True, print warnings about situations that may result in a crash
+    'millisecondsBetweenUpdates': 10,  # For ignoring frequent mouse event.
```

Comment:
I suggest `"mouseRateLimit": 30` or similar. the addition of "mouse" makes the parameter name more descriptive, and specifying the value in events per second should be more intuitive than milliseconds per event.